### PR TITLE
Add a site widget template

### PR DESCRIPTION
**Pull request by AdirAmsalem**
Add a template with a Site Widget.

The template contains the following application:

https://github.com/wix/cli-app-templates/assets/2660550/88ec545c-e9aa-4df8-8bf1-30cc05548d41

### CODE REVIEW

**Review by christopherbWIX — COMMENTED** [review]
Looks good. Few small comments.

File: chart-widget/template/src/site/widgets/custom-elements/chart/element.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "defaultWidth": 400,
+    "allowStretch": true
+  },
+  "height": {
+    "defaultHeight": 400
+  },
+  "installation": {
+    "autoAddToSite": true,
+    "essential": false,
+    "maxInstances": 0
```
Comment (AdirAmsalem) [inline chart-widget/template/src/site/widgets/custom-elements/chart/element.json:14]:
```suggestion
```

File: chart-widget/template/package.json.ejs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "dependencies": {
+    "@wix/design-system": "^1.0.0",
+    "@wix/editor": "^1.0.0",
+    "@wix/sdk-react": "^0.5.0",
+    "@wix/wix-ui-icons-common": "^3.0.0",
+    "react-to-webcomponent": "^2.0.0",
+    "recharts": "^2.0.0"
+  },
+  "devDependencies": {
+    "@types/react": "^16.0.0",
```
Comment (omrimn) [inline chart-widget/template/package.json.ejs:16]: we need to add `react-dom`
```suggestion
    "@types/react": "^16.0.0",
    "@types/react-dom": "^16.0.0",
```
But more templates needs this update, we can do a separate PR for that

**Review by omrimn — APPROVED** [review]
